src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -23,11 +26,16 @@ public class Intake extends SubsystemBase {
     private enum IntakeState {IDLE, MOTOR_START, WAITING, NOTE_ENTERING, NOTE_PAST_INTAKE, HAS_NOTE};
```

Comment:
I believe the state 'NOTE_ENTERING' is no longer used and can be dropped

Reply:
Yes. But no harm leaving it for now.